integration/v7/isolated/service_command_test.go

```diff
@@ -171,17 +171,16 @@ var _ = Describe("service command", func() {
 				})
 
 				It("displays the bound apps", func() {
-					//Delay to reduce flakiness
-					time.Sleep(10 * time.Second)
-					session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
-					Eventually(session).Should(Exit(0))
-
-					Expect(session).To(SatisfyAll(
-						Say(`Showing bound apps:\n`),
-						Say(`name\s+binding name\s+status\s+message\n`),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
-					))
+					Eventually(func(g Gomega) {
+						session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
+						g.Eventually(session).Should(Exit(0))
+						g.Eventually(session).Should(SatisfyAll(
+							Say(`Showing bound apps:\n`),
+							Say(`name\s+binding name\s+status\s+message\n`),
+							Say(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
+							Say(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
+						))
+					}).WithPolling(5 * time.Second).WithTimeout(3 * time.Minute).Should(Succeed())
```

Comment:
```suggestion
					}).Should(Succeed())
```

Reply:
I don't think we need to specify polling and timeout here as Eventually on line 174 has some built in

Reply:
Using .Wait() on the command will have it exit before continuing, so it can process the input and try again as needed

Reply:
What I mean is that After a failure if we retry 2 Milliseconds later there is a high probability that the status will not change.

Reply:
The majority of the Eventually that we use are done after an execution of a command to which we do not use the `.Wait()` function so it might make sense to have a polling interval that is lower than 5 seconds, but in cases where we want to retry to execution of the command itself, maybe it makes more sense to have a higher polling period, like in this case.
I think that we should configure the global timeouts and polling to the most common case in mind, but I believe the one from this comment is not it.

But if yll feel strongly about it we can create another PR where we do that change globally.

Reply:
If we .Wait() for the command execution to complete on line 175, then the polling is limited to the time it takes the command to finish executing and validate output before it immediately tries again. I don't think there's an issue there, CAPI can handle that request rate. This means when the command has the correct output we'll validate it quickly and continue. If we have polling every few seconds, that's a few seconds where the command might be ready to provide correct output but we haven't checked.